src/styles/card.css

```diff
@@ -0,0 +1,63 @@
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+}
+
+.card__name {
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--color-text-1);
+}
+
+.card__code {
+  line-height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-text-2);
+}
+
+.card__price {
```

Comment:
please reset default margin for **p** and and set indents for .card__block-price
![image](https://user-images.githubusercontent.com/54936044/232195489-a6b11c60-e770-4abd-b5f5-5cf59f9f0f42.png)